### PR TITLE
Created sample game struct using gamestate and actionmanagement functions

### PR DESCRIPTION
**Pull request by Tianlel**
We created a simple two-room (room1, room2) sample game struct with 1 player and 2 items (item_orb and item_table) and updated the Makefile for common.a.

room1 is to the left of room2. 
item_orb is placed in room1 and supports actions "TAKE", "DROP", and "PUT_ON".
item_table is placed in room2 and supports action "PUT_ON".

This game struct sample can be used by cli, action_management, and checkpointing for integration tests.

### CODE REVIEW

**Comment by Tianlel** [comment]
I have removed the files that I accidentally pushed to the repository.

The C file was suggested by @borjasotomayor in the last all-hands meeting. Many of the teams needed a sample game struct to test their functions and this issue was created to avoid duplication of sample game structs. This should be a quicker way to provide cli, action management and checkpointing a game struct to test their functions because the wdl team is still working on their load_wdl function #312.

File: src/common/src/sample_game.c [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    /* initialize items */
+    item_t *item_orb = item_new("orb", "magical orb", "an orb that can be held, placed, or tossed");
+    item_t *item_table = item_new("table", "a regular table", "this table can have things placed on it");
+    
+    /* introduce actions */
+    list_action_type_t *head = get_supported_actions();
+    action_type_t *actiontype_take = search_supported_actions(head, "TAKE");
+    action_type_t *actiontype_drop = search_supported_actions(head, "DROP");
+    action_type_t *actiontype_puton = search_supported_actions(head, "PUT_ON");
```
Comment (ecoble) [inline src/common/src/sample_game.c]: I have requested a change from action management that "PUT_ON" be changed to "PUT" so it reads better as user input, not sure if that change will go through but if it does watch out for it.

Comment (Tianlel) [reply src/common/src/sample_game.c]: Gotcha

Comment (mpcsh) [reply src/common/src/sample_game.c]: @ecoble @Tianlel that would be AM's responsibility to change (if you update a command, you also have to update all callers)

**Comment by Tianlel** [comment]
Closing statement:
This simple game struct is created with gamestate and actionmanagment functions and will be functional as soon as game state add add_function to master.

**Comment by lydiafilipe** [comment]
Issue Score: ✓+

Comments:
Nice work!

**Pull request merged**